NuRadioReco/framework/hybrid_shower.py

```diff
@@ -0,0 +1,16 @@
+import NuRadioReco.framework.base_shower
+
+class HybridShower(NuRadioReco.framework.base_shower):
```

Comment:
Hi, 
when inheriting from the base_shower this means, that only showerParameters can be stored in the parameter storage. Is that really feasible to cover all parameter for possible hybrid shower? I am not sure about this. However otherwise it might just be to dirty to not do it the same way. What do you think?
Cheers

Reply:
I expect the overlap between the two shower types to be very large. If some of the porperties are only used for one shower type, that is fine.
The hybrid shower class is only meant to store general shower properties, anything experiment-specific is supposed to go into a hybridDetector class